src/main/scala/com/mwz/sonar/scala/scoverage/ScoverageSensor.scala

```diff
@@ -182,13 +182,15 @@ private[scoverage] abstract class ScoverageSensorInternal extends Sensor {
   }
 }
 
-object ScoverageSensorInternal {
-  private val SensorName = "Scoverage Sensor"
-  private val DeprecatedScoverageReportPathPropertyKey = "sonar.scoverage.reportPath"
-  private val ScoverageReportPathPropertyKey = "sonar.scala.scoverage.reportPath"
-  private def getDefaultScoverageReportPath(scalaRawVersion: String) = {
+private[scoverage] object ScoverageSensorInternal {
+  val SensorName = "Scoverage Sensor"
+  val DeprecatedScoverageReportPathPropertyKey = "sonar.scoverage.reportPath"
+  val ScoverageReportPathPropertyKey = "sonar.scala.scoverage.reportPath"
+
+  def getDefaultScoverageReportPath(scalaRawVersion: String): Path = {
     // remove the extra part of the scala version, e.g 2.11.0 -> 2.11
+    // TODO: We could use a sem ver parsing here in case someone e.g. skips the patch version.
```

Comment:
:+1:, Absolutely, additionally someone could be using a beta release of scala or something different.

Maybe scaliform already have this implemented?

Reply:
Yes, quite possibly there is already something in scaliform...
I'll open another issue to look at this properly.